webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -2786,13 +2789,19 @@ class OntologyV2R2RSpec extends R2RSpec {
   "create a class with two cardinalities, use one in data, and allow only removal of the cardinality for the property not used in data" in {
     // Create a class with no cardinalities.
 
+    val label = LangString.make(LanguageCode.en, "A Blue Free Test class").fold(e => throw e.head, v => v)
+    val comment = Some(
```

Comment:
Could you actually implement value objects for `ClassLabel` and `ClassComment`?

Reply:
maybe I make this a follow-up task?